src/components/NcRichText/NcRichText.vue

```diff
@@ -217,15 +223,15 @@ export default {
 					},
 					prefix: false,
 				})
-				.processSync(this.useMarkdown
-				// In order to correctly show newlines in Markdown,
-				// each newline contains a non-breaking space
-					? this.text.slice()
-						.replace(/\n>\n/g, '\n>\u00A0\n')
-						.replace(/\n{2,}/g, (match) => {
-							return '\n' + '\n\u00A0\n'.repeat(match.length - 1)
-						})
-					: this.text)
+				.processSync(this.text.slice()
```

Comment:
What is the purpose of `slice` here?

Reply:
I thought `replace` is the mutating method 🙈

Reply:
strings are immutable in js :P